sky/utils/schemas.py

```diff
@@ -655,6 +655,7 @@ class RemoteIdentityOptions(enum.Enum):
     """
     LOCAL_CREDENTIALS = 'LOCAL_CREDENTIALS'
     SERVICE_ACCOUNT = 'SERVICE_ACCOUNT'
+    SKIP = 'SKIP'
```

Comment:
Open to better names, if any

Reply:
How about `remote_identity: null`?

Reply:
Being explicit makes it more clear to me, how about 'NO_UPLOAD'?

Reply:
Thanks, renamed to `NO_UPLOAD`